account/client.go

```diff
@@ -318,7 +318,7 @@ func (this *ClientImpl) getAccount(accData *AccountData, passwd []byte) (*Accoun
 	addr := types.AddressFromPubKey(publicKey)
 	scheme, err := s.GetScheme(accData.SigSch)
 	if err != nil {
-		return nil, fmt.Errorf("signature scheme error:%s", err)
+		return nil, fmt.Errorf("Signature scheme error: %s", err)
```

Comment:
we use lowercase to be consistent with golang std lib convention.

Reply:
Ah ok

Reply:
@laizy should be good now